### PR TITLE
ci: Automate Releases and publish artifacts

### PR DESCRIPTION
**Pull request by chris-fo**
## Description

Automate releases on push to `develop` and append prebuilt release artifacts.
Jira: https://jwplayer.atlassian.net/browse/BC-254

This PR solves #147.

### Steps completed:

According to our definition of done, I have completed the following steps:

- [ ] User stories met
- [ ] Storybook stories
- [ ] Unit tests added
- [ ] Linting passing
- [ ] Unit tests passing
- [ ] Docs updated (including config and env variables)
- [ ] Translations added
- [ ] UX tested
- [ ] Browsers / platforms tested
- [ ] Rebased & ready to merge without conflicts
- [ ] Reviewed own code

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+          github-token: ${{ secrets.github_token }}
+          output-file: "false"
+      - name: Install deps
+        run: yarn
+      - name: Build App
+        env:
+          RELEASE_VERSION: ${{ steps.changelog.outputs.tag }}
+        run: |
+          yarn build
+          tar czf build-${RELEASE_VERSION}.tar.gz ./build
```
Comment (budzworthy) [inline .github/workflows/release.yml]: Let's add `ott-web-app-build-` to the names here

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      - develop
+
+jobs:
+  build-and-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Generate changelog
```
Comment (budzworthy) [inline .github/workflows/release.yml:18]: This should happen in the existing bump-version workflow already. Maybe it makes sense to combine the 2 and rename it to create-new-release-version or something?

**Review by budzworthy — COMMENTED** [review]
This is really nice! I added just a few suggestions.  Otherwise, the output of the build looks a little off, but that is sort of separate from the github action part of things (depends on yarn and the FE build setup.)  So we can test and fix that separately.

File: .github/workflows/release.yml (hunk 1 above)
Comment (chris-fo) [reply .github/workflows/release.yml]: Done

File: .github/workflows/release.yml (hunk 2 above)
Comment (chris-fo) [reply .github/workflows/release.yml:18]: Done

File: .github/RELEASE_BODY_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Prebuilt artifacts
+
+`build.zip` and `build.tar.gz` contain the prebuilt JWP OTT-Webapp.
```
Comment (emptyflash) [inline .github/RELEASE_BODY_TEMPLATE.md]: Should this be `ott-web-app-build-{version}.tar.gz` and `ott-web-app-build-{version}.zip` just for extra clarity?

**Review by emptyflash — APPROVED** [review]

**Comment by budzworthy** [comment]
Good news!  The output works fine to serve the web app when put on a static file server.  I'm adding a commit to tweak the zip directory structure, then I think we can merge this.

File: .github/RELEASE_BODY_TEMPLATE.md (hunk 3 above)
Comment (budzworthy) [inline .github/RELEASE_BODY_TEMPLATE.md]:
```suggestion
`ott-web-app-build-<version>.zip` and `ott-web-app-build-<version>.tar.gz` contain the prebuilt JWP OTT-Webapp.
```

**Review by budzworthy — APPROVED** [review]

**Pull request merged**